c10/util/Logging.cpp

```diff
@@ -92,13 +92,20 @@ void SetAPIUsageLogger(std::function<void(const std::string&)> logger) {
   *GetAPIUsageLogger() = logger;
 }
 
-void LogAPIUsage(const std::string& event) {
-  (*GetAPIUsageLogger())(event);
+void LogAPIUsage(const std::string& event) try {
```

Comment:
It'll be more portable if you add the top level braces :)

Reply:
Are you saying avoid function-try-block? Cause that has been around for 10+ years, I doubt the templates for dispatch even compile on compilers old enough to no have function-try-block.

Reply:
oh TIL